.drone.yml

```diff
@@ -23,6 +23,16 @@ services:
     environment:
       - MYSQL_ALLOW_EMPTY_PASSWORD=yes
       - MYSQL_DATABASE=drupal
+  selenium:
```

Comment:
I don't have a selenium image in my local and the tests still passed so I don't think we need this, selenium we usually need for javascript tests and behat, which we dont have